ordered_product/tests/test_ordered_product_page.py

```diff
@@ -0,0 +1,278 @@
+        )
+        assert delivery_location0.location != saved_client0.area
+        assert delivery_location0 not in delivery_location_by_objects_filter
+
+        delivery_location_by_the_web_view = response.context['deliveries']
+        assert list(delivery_location_by_the_web_view) == list(delivery_location_by_objects_filter)
+
+    @pytest.mark.django_db()
+    def test_invalid_find_delivery(self, client, invalid_supplier_product_id, client0_login):
+        with pytest.raises(ObjectDoesNotExist):
+            SupplierProduct.objects.get(supplier_product_id=invalid_supplier_product_id)
+        response = client.get('/ordered_product/new_order/find_delivery/'+invalid_supplier_product_id)
+        assert response.status_code == 302
+        assert response['Location'] == '/client/'
+
+    @pytest.mark.django_db()
+    def test_valid_order_summary(
+        self,
+        client,
+        supplier_product0,
+        delivery_location0,
+        client0_login
+    ):
+        supplier_product0.save()
+        delivery_location0.save()
+        supplier_product_by_objects_filter = SupplierProduct.objects.get(
+            supplier_product_id=supplier_product0.supplier_product_id
+        )
+        delivery_location_by_objects_filter = DeliveryLocation.objects.get(
+            id=delivery_location0.id
+        )
+
+        assert supplier_product0 == supplier_product_by_objects_filter
+        assert delivery_location0 == delivery_location_by_objects_filter
+
+        url = (
+            '/ordered_product/new_order/order_summary/'
+            + str(supplier_product0.supplier_product_id)
+            + ','
+            + str(delivery_location0.id)
+        )
+        response = client.get(url)
+        assert response.status_code == 200
+
+        template_names = set(
+            template.origin.template_name for template in response.templates
+        )
+        assert 'ordered_product/new_order.html' in template_names
+
+        supplier_products_by_the_web_view = response.context['supplier_product']
+        delivery_location_by_the_web_view = response.context['delivery_location']
+        assert supplier_products_by_the_web_view == supplier_product_by_objects_filter
+        assert delivery_location_by_the_web_view == delivery_location_by_objects_filter
+
+    @pytest.mark.django_db()
+    def test_invalid_order_summary(
+        self,
+        client,
+        invalid_delivery_location_id,
+        invalid_supplier_product_id,
+        client0_login
+    ):
+        with pytest.raises(ObjectDoesNotExist):
+            SupplierProduct.objects.get(supplier_product_id=invalid_supplier_product_id)
+        with pytest.raises(ObjectDoesNotExist):
+            DeliveryLocation.objects.get(id=invalid_delivery_location_id)
+        url = (
+            '/ordered_product/new_order/order_summary/'
+            + str(invalid_supplier_product_id)
+            + ','
+            + str(invalid_delivery_location_id)
```

Comment:
Same

Reply:
The problem is the line longer then 120. I did as @yftacherzog said.